### DISCUSSION TITLE
Separate flows to NEL/data from one flowfile

### DISCUSSION THREAD

**Discussion by chpoqxie** [phaag/nfdump#425] [Q&A] [answered]
i have a one flow from server in which i receive a simple dataflows and a NEL flows
i need to split NEL flows from dataflows to storage on disk

i think this requires applying the following filters:
- to separate NEL flows 
nfdump -r nfdata.common -j -w nfdata.nel not asa event ignore
- to separate data flows
nfdump -r nfdata.common -j -w nfdata.flows asa event ignore

is this right? or should it be done in some other way?

ps: 1.7.1 latest from github.

**Comment by phaag** [comment, marked as answer]
Yes - that's the best way to do it.
